CMakeLists.txt

```diff
@@ -258,3 +260,36 @@ install(EXPORT TrantorTargets
         DESTINATION "${INSTALL_TRANTOR_CMAKE_DIR}"
         NAMESPACE Trantor::
         COMPONENT dev)
+
+# Doxygen documentation
+find_package(Doxygen OPTIONAL_COMPONENTS dot dia)
+if(DOXYGEN_FOUND)
+  set(DOXYGEN_PROJECT_BRIEF "Non-blocking I/O cross-platform TCP network library, using C++14")
+  set(DOXYGEN_OUTPUT_DIRECTORY docs/${PROJECT_NAME})
```

Comment:
Should we add the `docs` to the .gitignore?

Reply:
No, it is generated in ${CMAKE_CURRENT_BINARY_DIR}

Reply:
Well, if you do not generate in a separate directory, then yes.